tests/synthetic/test_step_function.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module tests functionality of the synthetic eye gaze step function.
+"""
+
+
+from __future__ import annotations
```

Comment:
we decided to not type annotated tests -- please change

Reply:
done